code/modules/mob/living/bot/secbot.dm

```diff
@@ -154,17 +156,13 @@
 			declare_arrests = !declare_arrests
 	attack_hand(usr)
```

Comment:
Delete this line. It creates infinite recursion.

Reply:
I replaced the attack_hand procs under Topic() with open_interface(usr)
That seems to have done it now. All looks well!
Thanks!

Reply:
Perfect! Glad I could help